package.json

```diff
@@ -1,29 +1,85 @@
 {
   "name": "etherspot-ui",
-  "version": "1.0.0",
+  "version": "0.1.0",
   "description": "An etherspot UI package",
-  "main": "index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "browser": "dist/index.cjs.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "test": "jest __tests__ --silent"
+    "test": "jest __tests__ --silent",
+    "test:watch": "jest __tests__ --watch",
+    "prepare": "npm run rollup:build",
+    "postinstall": "rm -rf node_modules/react node_modules/react-dom",
+    "rollup:build": "NODE_OPTIONS=--max-old-space-size=8192 rollup -c",
+    "rollup:watch": "rimraf ./node_modules/react ./node_modules/react-dom && rollup -c -w",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
   },
   "keywords": [],
   "author": "Etherspot Team",
   "license": "MIT",
   "dependencies": {
-    "@etherspot/transaction-kit": "^0.6.11",
-    "ethers": "^5.4.0",
-    "lodash": "^4.17.21"
+    "@etherspot/transaction-kit": "0.6.11",
+    "@tippyjs/react": "^4.2.6",
+    "autoprefixer": "^10.4.13",
+    "ethers": "5.4.0",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.4",
+    "react-content-loader": "^6.2.1",
+    "react-icons": "^4.4.0",
+    "react-ios-switch": "^0.1.19",
+    "react-router-dom": "^6.15.0",
+    "react-spinners": "^0.13.2",
+    "reflect-metadata": "^0.1.13",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rxjs": "^6.6.7",
+    "styled-components": "^5.3.5"
```

Comment:
Please version lock these dependencies. Remember when you're building a library, we need to provide stability as a guarantee to the other developers